nipype/algorithms/modelgen.py

```diff
@@ -144,6 +144,61 @@ def scale_timings(timelist, input_units, output_units, time_repetition):
     timelist = [np.max([0., _scalefactor * t]) for t in timelist]
     return timelist
 
+def bids_gen_info(bids_event_files,
+    condition_column='trial_type',
```

Comment:
The space before each parameter should align with the open paren:
```suggestion
                  condition_column='trial_type',
```